docs/facts/faq.rst

```diff
@@ -21,8 +21,10 @@ I have multiple CPU
 
 Simply use the :code:`-m | --multiprocess` argument to activate
 the usage of multiple processes.
-You can in addition to that use the :code:`-p | --processes`
-argument to tell set the number of simultaneous process to run.
+You should in addition to the :code:`-m` specify the :code:`-p | --processes`
+argument as the default value is 25 simultaneous processes. This number will 
```

Comment:
If `-p` is avoided, we follow the number of available CPU cores @spirillen (Documenting will fix #78)

Reply:
Have rewritten the sentence(s)

Have a second look :smile: